auth/presentation/src/main/java/com/nibalk/tasky/auth/presentation/login/LoginScreen.kt

```diff
@@ -64,26 +61,35 @@ private fun LoginScreen(
     TaskyBackground(
         title = stringResource(id = R.string.auth_welcome),
         footer = {
-            LoginScreenFooter(onAction)
+            if (WindowInsets.ime.getBottom(LocalDensity.current) <= 0) {
+                LoginScreenFooter(onAction)
+            }
         }
     ) {
         // Email Field
         Spacer(modifier = Modifier.height(MaterialTheme.spacing.spaceMedium))
         TaskyTextField(
             state = state.email,
-            endIcon = if (state.isValidEmail) {
+            endIcon = if (state.emailError == null) {
                 CheckMarkIcon
             } else null,
             hint = stringResource(id = R.string.auth_email),
             modifier = Modifier.fillMaxWidth(),
             keyboardType = KeyboardType.Email,
-            error = if(!state.isValidEmail) {
-                stringResource(
-                    id = R.string.auth_must_be_a_valid_email,
-                    AuthDataValidator.PASSWORD_MIN_LENGTH
-                )
-            } else {
-                null
+            error = when (state.emailError) {
+                AuthDataValidateError.EmailError.EMPTY -> {
+                    stringResource(
+                        id = R.string.auth_must_enter_email,
+                        AuthDataValidator.PASSWORD_MIN_LENGTH
```

Comment:
Seems odd to me to pass in the password min length for the email, is this intentional?